libs/deal-ticket/src/components/deal-ticket/deal-ticket-stop-order.tsx

```diff
@@ -332,7 +344,7 @@ const Size = ({
         const { value, ...props } = field;
         const id = `order-size${oco ? '-oco' : ''}`;
         return (
-          <div className="mb-4">
+          <div className={`mb-${isLimitType ? '4' : '2'}`}>
```

Comment:
I'm pretty  sure this won't compile.. you'll get away with it as we definitely already use mb-2 and mb-4, but dynamically creating classes like this won't work with tailwind

```suggestion
          <div className={isLimitType ? 'mb-4' : 'mb-2'}>
```